napari/components/_layers/view/main.py

```diff
@@ -128,17 +125,15 @@ def dropEvent(self, event):
 
 
 class QtDivider(QFrame):
-    unselectedStlyesheet = "QFrame {;}"
-    selectedStlyesheet = """QFrame {border: 2px solid %s;
-        background-color: white; border-radius: 3px;}""" % palette['text']
-
     def __init__(self):
         super().__init__()
         self.setSelected(False)
         self.setFixedSize(50, 2)
 
     def setSelected(self, bool):
         if bool:
```

Comment:
`bool` is the built in boolean type. I think we should use a different variable name here.